### PR TITLE
Fixed List and spacing, fixes issues #919

### PR DESCRIPTION
**Pull request by nemaniarjun**

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
-    for C, C++ and other Clang supported languages.)
+-  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function for C, C++ and other Clang supported languages.)
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: nono this is not what the issue wants i think, it simply wants you to allign the ``for`` with the first double backtick.

also the hyphens need to be at the same level.

what you did was move all text on the same line, we dont want that.

Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: also please commit message: ``fixes``->``Fixes``

**Comment by Adrianzatreanu** [comment]
it appears that the 2 commits do the absolute same thing, could you remove them? also please rebase correctly, as we dont want merge commits. for more information, use ``cobot explain rebase`` on gitter :) https://gitter.im/coala/coala

**Comment by sils** [comment]
Hey,

here's a rebase tutorial.

---

Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

(Be sure to use coala-bears on the coala-bears repository :))

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by sils** [comment]
(but first reopen your PR otherwise this won't work)

**Pull request closed without merging**